provider/datagen/src/transform/cldr/time_zones/mod.rs

```diff
@@ -92,13 +93,18 @@ macro_rules! impl_data_provider {
 
             impl IterableDataProvider<$marker> for TimeZonesProvider {
                 fn supported_locales(&self) -> Result<Vec<DataLocale>, DataError> {
-                    Ok(self
-                        .source
-                        .cldr()?
-                        .dates("gregorian")
-                        .list_langs()?
-                        .map(DataLocale::from)
-                        .collect())
+                    if <$marker>::KEY == MetaZonePeriodV1Marker::KEY {
```

Comment:
Note: Actually this code here is the only part strictly necessary. The rest is cleanup.

Reply:
Great! Good to know that!